source/blender/makesrna/intern/rna_space.c

```diff
@@ -480,6 +480,7 @@ static const EnumPropertyItem buttons_context_items[] = {
      "Bone Constraints",
      "Bone Constraint Properties"},
     {BCONTEXT_MATERIAL, "MATERIAL", ICON_MATERIAL, "Material", "Material Properties"},
+    {BCONTEXT_GAME, "GAME", ICON_SCRIPT, "Game", "Game Object Properties"},
```

Comment:
This is not really important, but Game Object properties suggests that there is an existing gameobject in the viewport whereas Gameobjects are only created after bge start. I'd rename this Object Game Components and Properties or something shorter

Reply:
Personally, I think it wouldn't be too confusing since it's not rare to find such settings in Blender UI that are only taken into effect when the game or animation is running. If it's ok to call it a "component panel" when `KX_PythonComponent` instance is only created in runtime, I suppose it wouldn't be too unreasonable to expect users to understand that a "game object" settings wouldn't imply that there's a live `KX_GameOjbect` somewhere inside the editor.

That being said, I don't object to rename it if there's a better alternative than "Game Object Properties". Maybe "Game Scripts" could be an option? (By the way, if I manage to implement KX_GameObject related stuffs as discussed on Discord, they will likely end up in the same panel too.)